thoth_user_api/swagger.yaml

```diff
@@ -680,6 +680,90 @@ paths:
           schema:
             $ref: "#/definitions/AnalysisResponseError"
 
+  /dependency-monkey/python:
+    post:
+      tags: [Dependency Monkey]
+      x-swagger-router-controller: thoth_user_api.api_v1
+      operationId: post_dependency_monkey_python
+      summary: Run Dependency Monkey on the given application stack.
+      parameters:
+        - name: application_stack
+          in: body
+          required: true
+          description: Specification of Python application stack.
+          schema:
+            $ref: "#/definitions/PythonStack"
+        - name: runtime_environment
+          in: query
+          description: >
+            Runtime environment in which the given stack will be run.
+          type: string
+          required: false
+        - name: debug
+          in: query
+          required: false
+          description: >
+            Run the given analysis in a verbose mode so developers can debug it.
+          type: boolean
+          default: false
+      responses:
+        202:
+          description: The Dependency Monkey analysis is scheduled.
+          schema:
+            $ref: "#/definitions/AnalysisResponse"
+        400:
+          description: On invalid request.
+          schema:
+            $ref: "#/definitions/AnalysisResponseError"
+
+  /dependency-monkey/python/{analysis_id}/log:
```

Comment:
maybe we should rename 'analysis_id' to a more generic 'job_id' all over the place?!